app/formatters/duty_expression_formatter.rb

```diff
@@ -77,14 +81,21 @@ def format(opts = {})
           output << monetary_unit
         end
         if measurement_unit_abbreviation.present?
-          output << if opts[:formatted]
+          output << if formatted
                       "/ <abbr title='#{measurement_unit.description}'>#{measurement_unit_abbreviation}</abbr>"
```

Comment:
could use `helper.tag.abbr(measurement_unit_abbreviation, title: measurement_unit.description)` - handles HTML escaping itself then rather than the implicit trust of date with `.html_safe`